shelley/chain-and-ledger/formal-spec/delegation.tex

```diff
@@ -662,10 +666,15 @@ \subsection{Stake Pool Rules}
       \item The key is added to the set of registered stake pools.
       \item The pool's parameters are stored.
     \end{itemize}
-  \item Stake pool parameter updates are handled by \cref{eq:pool-rereg}.
-    This rule, which also matches on the certificate type $\type{DCertRegPool}$,
-    is distinguished from \cref{eq:pool-reg} by the requirement that
-    the pool be registered. This rule also ends stake pool retirements.
+  \item Stake pool parameter updates are handled by \cref{eq:pool-rereg} and \cref{eq:pool-rereg-late}.
+    These rules, which also matches on the certificate type $\type{DCertRegPool}$,
+    are distinguished from \cref{eq:pool-reg} by the requirement that the pool be registered.
+    In the case that the certificate is received before the last $\StabilityWindow$-many slots
+    of the epoch, it is immediately included in $\var{poolParam}$.
+    Otherwise, it is added to the future pool parameters $\var{fPoolParam}$,
+    which will override $\var{poolParam}$ with the new values in the $\mathsf{TICK}$ transition
+    (see Figure~\cref{fig:rules:epoch}).
```

Comment:
I think it would be very nice to explain why we do this, otherwise it's a bit confusing!

Reply:
great point, thanks!